test/tests/functional/pbs_cray_smoketest.py

```diff
@@ -157,19 +158,17 @@ def test_cray_compute_job(self):
                            id=vname)
         # Sleep for some time before aprun actually starts
         # using the reservation
-        time.sleep(2)
+        time.sleep(6)
```

Comment:
Why did you have to increase the sleep time?  Does it really take this long for the ALPS information to be updated?

Reply:
Now that I have changed the job script which now has a 'sleep 5' before doing 'aprun -B sleep 10', I wanted to wait 6 seconds before checking the apstat output.

Reply:
Add a self.logger() in front of this saying "Sleeping 6 seconds for ..."